src/vorta/views/main_window.py

```diff
@@ -163,6 +182,60 @@ def profile_add_action(self):
         window.profile_changed.connect(self.profile_add_edit_result)
         window.rejected.connect(lambda: self.profileSelector.setCurrentIndex(self.profileSelector.currentIndex()))
 
+    def profile_export_action(self):
+        window = ExportWindow(profile=self.current_profile)
```

Comment:
Suggest to use `self.profile()` here as reference to the profile saved in the DB. Else updated settings won't be saved during export.

Reply:
For that to work in need to add BackupProfileMixin to the main window. Shall I do it?

Reply:
I added it now and it works. But I can change it if you had something different in mind.